cmd/agent/launcher/launcher.c

```diff
@@ -0,0 +1,29 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+
+#ifndef DD_AGENT_PATH
+#define DD_AGENT_PATH ""
```

Comment:
Since this value will systematically fail at runtime, shouldn't you error out at build time in this case?

Reply:
You're right. Changing it